source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -42,7 +43,8 @@ export default class PinCode extends Component<Props> {
     if (
       !Object.prototype.hasOwnProperty.call(value, key) ||
       value[key] === '' ||
-      inputNewValue === ''
+      inputNewValue === '' ||
+      (inputNewValue !== '' && inputNewValue.length === 1)
```

Comment:
I think `inputNewValue === '' || inputNewValue.length === 1` is sufficient here because if the string has a length of 1 it's definitely not "empty string"

Reply:
Improved @DominikGuzei